appinfo/routes.php

```diff
@@ -11,6 +11,7 @@
 		['name' => 'config#setSensitiveUserConfig', 'url' => '/config/sensitive', 'verb' => 'PUT'],
 		['name' => 'config#setAdminConfig', 'url' => '/admin-config', 'verb' => 'PUT'],
 		['name' => 'config#setSensitiveAdminConfig', 'url' => '/admin-config/sensitive', 'verb' => 'PUT'],
+		['name' => 'config#autoUpdateConfig', 'url' => '/admin-config/auto-update', 'verb' => 'POST'],
```

Comment:
Nitpick: We could rename that to something more explicit like "autoDetectAvailableEndpoints" or "autoDetectAvailableFeatures".